src/amo/pages/AddonReviewList/index.js

```diff
@@ -333,29 +333,36 @@ export class AddonReviewListBase extends React.Component<InternalProps> {
           {reviewId && (
             <FeaturedAddonReview addon={addon} reviewId={reviewId} />
           )}
-          <CardList
-            className="AddonReviewList-reviews-listing"
-            footer={paginator}
-            header={reviewCountHTML}
-          >
-            <ul>
-              {allReviews.map((review, index) => {
-                // TODO: Remove this and use the API to filter out the featured review once
-                // https://github.com/mozilla/addons-server/issues/9424 is fixed.
-                if (
-                  !reviewId ||
-                  (review && review.id.toString() !== reviewId)
-                ) {
-                  return (
-                    <li key={String(index)}>
-                      <AddonReviewCard addon={addon} review={review} />
-                    </li>
-                  );
-                }
-                return null;
-              })}
-            </ul>
-          </CardList>
+          {// TODO: remove this condition after
```

Comment:
This is totally fine as is but you can also put comments in JSX like `{/* some long comment */}`, FYI.